strongbox-client/src/main/java/org/carlspring/strongbox/service/impl/ProxyRepositoryConnectionPoolConfigurationServiceImpl.java

```diff
@@ -129,7 +129,7 @@ public void setMaxPerRepository(String repository,
         }
         else
         {
-            LOGGER.warn("Not setting max repository connections to %s as it is no positive value", max);
+            LOGGER.warn("Not setting max repository connections to {} as it is no positive value", max);
```

Comment:
All our loggers are actually lowercase. Would you mind checking the rest of the code aross the pull request for similar ones? Thanks! :)
```suggestion
            LOGGER.warn("Not setting max repository connections to {} as it is no positive value", max);
```